src/packages/contracts/config/index.ts

```diff
@@ -37,6 +38,10 @@ import pancakePairV2Abi from './externalAbis/PancakePairV2.json';
 import vBnbAbi from './externalAbis/VBnb.json';
 import XsequenceMulticall from './externalAbis/XsequenceMulticall.json';
 
+// bscmainnet.json file is too big and TS does not parse its type properly
+const venusProtocolBscMainnetDeployments =
+  venusProtocolBscMainnetDeploymentsJson as typeof venusProtocolBscTestnetDeployments;
```

Comment:
The issue is that the bigger this document get, the more processing power will be used during the build process and so we might reach a bottleneck.
I think it's a good time to refactor this to use a library that would read the JSON file in chunks (e.g.: https://www.npmjs.com/package/JSONStream). Let me know if you want to collaborate on that.

Reply:
Ok, I'll be looking into it!